test/user-profile/helpers.unit.spec.js

```diff
@@ -32,13 +33,13 @@ const schemaToConfigIds = {
   '22-5490': '5490',
   '22-5495': '5495',
   '40-10007': '40-10007',
+  VIC: 'VIC',
   definitions: 'N/A'
 };
 
 const excludedForms = new Set([
   '28-1900',
-  '28-8832',
-  'VIC'
+  '28-8832'
```

Comment:
Should we consider using the metalsmith config's ignorelist to keep this list current?

Reply:
That assumes that we wouldn't launch a form without SiP, which is maybe true, but not sure I want to require us to do that.